docs/index.html

```diff
@@ -89,9 +89,37 @@
         placeholder: 'Search...'
       },
 
-      'flexible-alerts': {
-        style: 'callout'
-      },
+      // Add languages here for message box translations
+      'flexibleAlerts': {
+          note: {
+            label: {
+              '/i18n/chinese/': '笔记',
```

Comment:
```suggestion
              '/i18n/chinese/': '注意',
```

Adding Miya's comment.